packages/core/src/domain/session/oldCookiesMigration.ts

```diff
@@ -37,7 +37,7 @@ export function tryOldCookiesMigration(options: CookieOptions) {
 
     if (!isSessionInExpiredState(session)) {
       session.expire = String(dateNow() + SESSION_EXPIRATION_DELAY)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏not necessarily for this PR, but it could be nice to extract that as `expandSession` in `sessionState` and reuse it in `sessionStoreOperations` as well

Reply:
👍